.github/workflows/ci.yml

```diff
@@ -84,11 +86,33 @@ jobs:
         uses: google-github-actions/setup-gcloud@v1.0.1
       - name: gcloud functions deploy
         run: |
-          gcloud functions deploy notification-ci \
+          gcloud functions deploy extension-ci \
             --region europe-west3 \
             --source sources \
-            --entry-point notificationTrigger \
+            --entry-point extensionTrigger \
             --runtime nodejs16 \
             --trigger-http \
             --allow-unauthenticated
       - run: npm run test
+
+  webhook_test:
+    name: Webhook test
+    concurrency: Webhook Test for Notification Module
+    needs: extension_build
+    runs-on: ubuntu-latest
+    env:
+      ADYEN_INTEGRATION_CONFIG: ${{ secrets.ADYEN_INTEGRATION_CONFIG }}
+      CI_NOTIFICATION_URL: ${{ secrets.CI_NOTIFICATION_URL }}
+    defaults:
+      run:
+        working-directory: notification
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
+      - run: npm ci
+      - run: npx lockfile-lint --path package-lock.json --allowed-schemes "https:" "git+https:" --allowed-hosts npm github.com
+      - run: npm run check
```

Comment:
Do we need the below commands here again?

- npm ci
- npm run check

Since they are already built and run in the above build(Notification_build).

Reply:
`npm ci` is needed to install package into node_modules folder, so do the update of package-lock.json

https://github.com/commercetools/commercetools-adyen-integration/pull/1061/commits/fd73e270698d0fcf6e1432f790b56aaccbd982e6